webxrlayers-1.bs

```diff
@@ -0,0 +1,730 @@
+[=XR Compositor=] and is expected to be representative of the scene rendered into the layer.
+
+XRQuadLayer {#xrquadlayertype}
+-----------
+A XRQuadLayer renders a layer that takes up a flat rectangular space in the virtual environment.
+Only the front of the layer must be visible; the back face must not be drawn by the [=XR Compositor=].
+
+A XRQuadLayer has no thicknes. It is a two-dimensional object positioned and oriented in 3D space. The position 
+of a quad refers to the center of the quad.
+
+<pre class="idl">
+interface XRQuadLayer : XRLayer {
+  readonly attribute boolean stereo;
+  attribute XRReferenceSpace referenceSpace;
+  attribute XRRigidTransform transform;
+
+  attribute float width;
+  attribute float height;
+};
+</pre>
+
+The <dfn attribute for="XRQuadLayer">stereo</dfn> attribute returns if the layer is rendered in stereo. 
+
+The <dfn attribute for="XRQuadLayer">transform</dfn> attribute sets and returns the offset and orientation relative to the
+<dfn attribute for="XRQuadLayer">referenceSpace</dfn> attribute.
+
+The <dfn attribute for="XRQuadLayer">width</dfn> and <dfn attribute for="XRQuadLayer">height</dfn> attributes 
+set and return the width and height of the layer in meters.
+
+XRCylinderLayer {#xrcylinderayertype}
+---------------
+A XRCylinderLayer renders a layer that takes up a curved rectangular space in the virtual environment.
+Only the front of the layer must be visible; the back face must not be drawn by the [=XR Compositor=].
+
+A XRCylinderLayer has no thicknes. It is a two-dimensional object positioned and oriented in 3D space. The position 
+of the cylinder refers to the center of the quad.
+
+<pre class="idl">
+interface XRCylinderLayer : XRLayer {
+  readonly attribute boolean stereo;
+  attribute XRReferenceSpace referenceSpace;
+  attribute XRRigidTransform transform;
+
+  attribute float radius;
+  attribute float centralAngle;
+  attribute float aspectRatio;
+};
+</pre>
+
+The <dfn attribute for="XRCylinderLayer">stereo</dfn> attribute returns if the layer is rendered in stereo. 
+
+The <dfn attribute for="XRCylinderLayer">transform</dfn> attribute sets and returns the offset and orientation relative to the
+<dfn attribute for="XRCylinderLayer">referenceSpace</dfn> attribute.
+
+The <dfn attribute for="XRCylinderLayer">radius</dfn> attribute controls the radius in meters of the cylinder.
+
+The <dfn attribute for="XRCylinderLayer">centralAngle</dfn> attribute controls the angle in degrees of the visible section of the cylinder.
+It grows symmetrically around the 0 angle.
+
+The <dfn attribute for="XRCylinderLayer">aspectRatio</dfn> attribute controls the ratio of the visible cylinder section.
+
+<img src="images/cylinder_layer_params.png" style="width:80%"/>
+
+XREquirectLayer {#xrequirectlayertype}
+---------------
+A XREquirectLayer renders a layer where the [=XR Compositor=] must map an equirectangular coded data onto the inside of a sphere.
```

Comment:
```suggestion
An {{XREquirectLayer}} renders a layer where the [=XR Compositor=] must map an equirectangular coded data onto the inside of a sphere.
```
I'm gonna stop flagging these for now, but in general we want to make sure any interface names like this are properly linked.

Reply:
done